bgpd/bgp_evpn.c

```diff
@@ -6,6 +6,7 @@
 
 #include <zebra.h>
 
+#include "bgpd/bgp_addpath_types.h"
```

Comment:
let's try to keep the different include parts separate.  Put this with the rest of the bgp includes